### PR TITLE
Queues - Kaitlin Ramirez - Static-Site

### PR DESCRIPTION
**Pull request by arrozconcurry**
# Static Site
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| Did you have to resolve any issues when running the HTML Validator? If so, what were they?  |   Across my four html files I had a total of 7 errors. 3 in `about.html` and 4 `portfolio.html`.   I'm not sure how to change the errors as they would change the format of the website.  |
| Why is it important to consider and use semantic HTML? | For e-readers   |
| How did you decide to structure your CSS? |  I went top down, and then left right. So everything in `index.html` top down, then everything in `about.html` top down, etc. With the exception of the `footer` property. I have that at the bottom of the CSS because it felt right.  |
| What was the most challenging piece of this assignment? | I really wanted a drop down description for my portfolio. I dove deep into stack overflow to find a way to do it in CSS. The images move a little when you click it, but otherwise I'm happy with it.   |
| Describe one area that you gained more clarity on when completing this assignment | The Box Model! 😃   |

### CODE REVIEW

**Comment by droberts-sea** [comment]
# Static Site
## What We're Looking For

| Feature | Feedback    |
| :------------- | :------------- |
|  **Baseline** |   |
|   Appropriate Git Usage	|   yes	|
|   Answered comprehension questions	|   Semantic HTML also helps make your HTML easier for human programmers to read and maintain.	|
|   Page fully loads | yes |
|   No broken links (regular or images) | yes |
|   Includes at least 4 pages and styling | yes |
|  **HTML** |   |
|   Uses the high-level tags for organization: `header, footer, main` | Some. You're pretty good about using `<header>` and your pages don't have a footer so that doesn't apply, but I'd like to see you wrap the rest of the page outside of `<header>` in a `<main>` tag. |
|   Appropriately using semantic tags: `section, article`, etc. | yes |
|   All images include alternate text | yes |
|  **CSS** |   |
|   Using class and ID names in style declarations | yes |
|   Style declarations are DRY  | yes |
|  **Overall** |   |

Great work overall. Site is attractive and easy to navigate, and HTML is for the most part well-structured and easy to read.

Something you may have noticed is that CSS files can quickly get large and difficult to work with. There are two main strategies for dealing with this.

The first is to make frequent use of comments, to help the reader know which part of the page a group of rule-sets is supposed to be styling.

The second is to split out the CSS for different pages into different files, and then each page only links the styles they need. So the project setup might look like

```
.
├── hobby-blog.html
├── index.html
└── stylesheets/
    └── style.css
    └── hobby-blog.css
    └── index.css
```

and then in the individual HTML files:

```html
<!-- hobby-blog.html -->
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Hobby Blog</title>
  <link href="stylesheets/style.css" rel="stylesheet">
  <link href="stylesheets/hobby-blog.css" rel="stylesheet">
<!-- ... -->
```

File: website/stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  border-radius: 20px;
+  clear: both;
+}
+
+.project {
+  height: 180px;
+  width: auto;
+}
+
+#dms {
```
Comment (dlaguerta) [inline website/stylesheets/style.css:63]: A clearer id name would be useful here

File: website/stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
+html {
+    display: block;
+    background-color: #46494C;
+}
+
+body, h1, h2, h3, p {
+  margin: 0;
+  padding: 0; }
+
+h1 {
```
Comment (dlaguerta) [inline website/stylesheets/style.css:10]: Looks like font-family in H1 and H2 are duplicated: what might you do instead to keep the CSS file more DRY?

File: website/about.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          <h2>Name:</h2> <p>English first name, Indian middle name, Spanish last name</p>
+          <h2>Background:</h2> <p>Escaped Florida.</p>
+
+      </div>
+
+      <div>
+        <ul>
+          <h2>Born:</h2> <p>Escaped Texas.</p>
+          <h2>Interests:</h2>
+          <li>Eatting Chocolate Babka</li>
```
Comment (dlaguerta) [inline website/about.html:33]: spelling mistake: eating

File: website/portfolio.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  <nav class="pages">
+    <a href="index.html">Home</a>
+    <a href="about.html">About Me</a>
+    <a href="#">Portfolio</a>
+    <a href="code-journal.html">Blog</a>
+  </nav>
+
+  <section class="projects">
+    <h2>Front-End</h2>
+    <a href="#hide1" id="hide1"><img class="project" src="images/wannaada.png" alt="I Wanna Ada Cuz It Rocks" title="Click Me for info!"/></a>
```
Comment (dlaguerta) [inline website/portfolio.html:22]: Does "hide" and "show" attributes align how you want it to?

File: website/stylesheets/style.css [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+  padding: 0 70px;
+}
+
+#description1,
+#show1,
+#hide1:target {
+  display: none;
+}
+
+#hide1:target + #show1,
```
Comment (dlaguerta) [inline website/stylesheets/style.css:105]: Neat use of CSS here! Later on, you'll learn to use Javascript to do similar actions :)

**Review by dlaguerta — COMMENTED** [review]
Hi Kaitlin,
Your commit messages are clear, and you're committing often which is good! If you want to become more of a badass at git commits, check out this article: https://chris.beams.io/posts/git-commit/ (I just read this recently, and found it helpful for internships).
I'd challenge you to think about your design for accessibility: your website looks clean and well organized, but the color contrast between the background and your font size and colors makes it hard to read. Semantic tags are also important for that reason, and I'd encourage you to use them more! 
I've made a few inline comments on a few tags for further investigation.
Overall, nice job!

**Pull request closed without merging**